lighthouse-viewer/app/src/main.js

```diff
@@ -48,7 +48,11 @@ function main() {
   window.viewer = new LighthouseReportViewer();
 }
 
-main();
+// Defer main until the entire script is loaded.
+// This is because the current build system for the viewer concats files in alphabetical order,
+// which means "psi-api.js" comes _after_ "main.js", causing the constructor to error if `main`
+// is called synchronously.
+Promise.resolve().then(main);
```

Comment:
```suggestion
window.addEventListener('DOMContentLoaded', main);
```

Reply:
(and probably don't need the build system comment, this is a somewhat normal situation (see `report-template.html`, for instance))